packages/block-library/src/post-featured-image/edit.js

```diff
@@ -40,14 +42,11 @@ import DimensionControls from './dimension-controls';
 import OverlayControls from './overlay-controls';
 import Overlay from './overlay';
 import { useToolsPanelDropdownMenuProps } from '../utils/hooks';
+import { unlock } from '../lock-unlock';
+import { DEFAULT_MEDIA_SIZE_SLUG } from './constants';
```

Comment:
Suggestion: There's no reason to extract constant into a separate file if it's only used once. Let's just inline it.